tests/datadriven/test_variants.py

```diff
@@ -346,3 +347,22 @@ def convertPyvcfNumber(number):
         testMetaLength = (
             1 + len(self._formats) + len(self._infos) - gtCounter)
         self.assertEqual(len(keyMap), testMetaLength)
+
+    def testGetVariant(self):
+        variantSet = self._gaObject
+        for referenceName in self._referenceNames:
+            refnameVariants = self._getPyvcfVariants(referenceName)
+            for variant in refnameVariants:
+                md5 = self._hashVariant(variant)
+                compoundId = datamodel.VariantCompoundId(
+                    variantSet.getCompoundId(), referenceName,
+                    variant.start, md5)
+                gotVariant = variantSet.getVariant(compoundId)
+                self.assertEqual(str(compoundId), gotVariant.id)
```

Comment:
We need some negative tests here --- what happens when we try to get some variants that are very like this one? Off by one coords, slightly different ref and alt?